sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDAFSuite.scala

```diff
@@ -123,6 +133,26 @@ class HiveUDAFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
     }
   }
 
+  test("SPARK-27207: customized Hive UDAF with two aggregation buffers for Sort" +
```

Comment:
does this test fail without your patch? I think we can write a test with a custom UDAF which fails without initialization, but Hive UDAF does not fail without initialization.

Reply:
So it used to fail earlier without my patch, but your latest patch seems to have fixed it. Will come up with another test case. Thank you.